content/guides/integrations/memcached/Go.md

```diff
@@ -155,8 +150,8 @@ func main() {
 			sqrt := big.NewFloat(0).Sqrt(in)
 			out, _ := sqrt.MarshalText()
 
-                        // Pause for 3 milliseconds as a throttle to "avoid CPU saturation".
-                        <-time.After(3 * time.Millisecond)
+			// Pause for 3 milliseconds as a throttle to "avoid CPU saturation".
+			<-time.After(3 * time.Millisecond)
```

Comment:
Just curious why not using time.Sleep(3 * time.Milliseconds) as we're not in a select op?

Reply:
Just a preference, but sure, I'll make it a time.Sleep, thanks @basvanbeek!